scheduler.go

```diff
@@ -266,7 +267,7 @@ func (s *Scheduler) calculateMonths(job *Job, lastRun time.Time) nextRun {
 
 		return nextRunResult
 	}
-	next := lastRunRoundedMidnight.Add(job.getAtTime()).AddDate(0, job.interval, 0)
+	next := lastRunRoundedMidnight.Add(job.getFirstAtTime()).AddDate(0, job.interval, 0)
 	return nextRun{duration: until(lastRunRoundedMidnight, next), dateTime: next}
```

Comment:
It may be a bug here, should be `until(lastRun, next)`, right? but I can't find a way to make `len(job.daysOfTheMonth)==0` and get here, you want me to fix or delete it?

Reply:
Go ahead and fix it if you don't mind

Reply:
done.